backend/src/main/java/com/woowacourse/kkogkkog/infrastructure/application/PushAlarmListener.java

```diff
@@ -23,6 +26,12 @@ public void sendNotification(PushAlarmEvent pushAlarmEvent) {
         String accessToken = pushAlarmEvent.getBotAccessToken();
         String hostMemberId = pushAlarmEvent.getHostMemberId();
         String message = pushAlarmEvent.getMessage();
-        slackClient.requestPushAlarm(accessToken, hostMemberId, message);
+        if (pushAlarmEvent.isWoowacourseWorkspace()) {
+            woowacoursePushAlarmClient.requestPushAlarm(accessToken, hostMemberId, message);
+        }
+        if (pushAlarmEvent.hasBotAccessToken()) {
+            commonPushAlarmClient.requestPushAlarm(accessToken, hostMemberId, message);
+        }
+
     }
```

Comment:
현재 Member 테이블의 워크스페이스 정보가 woowacourse에 해당된다는 점을 어떻게 알 수 있나요? 
우테코 워크스페이스 사용자인지를 파악하려면 email을 사용해야 하지 않나요?

알림 두번 보내지 않도록 return해줘야 할 것 같고, 이상한 위치에 공백이 있어요.

```suggestion
        if (pushAlarmEvent.isWoowacourseWorkspace()) {
            woowacoursePushAlarmClient.requestPushAlarm(accessToken, hostMemberId, message);
            return;
        }
        if (pushAlarmEvent.hasBotAccessToken()) {
            commonPushAlarmClient.requestPushAlarm(accessToken, hostMemberId, message);
        }
    }
```

Reply:
아니면 woowacourse 워크스페이스의 사용자는 무조건 해당 워크스페이스의 정보만을 사용하도록 로그인 쪽 로직을 수정해야 할 것 같아요.

Reply:
> 우테코 워크스페이스 사용자인지를 파악하려면 우테코 크루의 email 목록과 사용자의 이메일 값을 비교해야 하지 않나요?
현재는 다른 워크스페이스 계정으로 로그인하면 해당 워크스페이스로 알림을 보냅니다. 정 피드백을 들어보니 리팩터링이 필요할 것 같네요.
1. 우선 email로 woowacourse 워크스페이스 사용자를 확인하여 메세지를 보낸다. 이후 다른 워크스페이스인지 확인한다.
2. woowacourse 이외의 워크스페이스로는 알림을 보내지 않는다.

제 생각은 1번은 적용하는게 좋을 것 같고 2번은 확장성이 제한된다는 단점이 있을 것 같은데 어떻게 생각하시나요??

Reply:
추가로 Member 부분을 보면서 알게 된 건데 지금 구현한건 최초에 우아코스로 가입하지 않으면 안되네요ㅜ